docs/components/integration-framework/connectors/out-of-the-box-connectors/aws-lambda.md

```diff
@@ -0,0 +1,49 @@
+---
+id: aws-lambda
+title: AWS Lambda Connector
+description: Invoke AWS Lambda function.
+---
+
+The AWS Lambda Connector allows you to connect your BPMN service with the Amazon Web Service's [AWS Lambda Service](https://aws.amazon.com/lambda/).
+
+## Prerequisites
+
+To use an **AWS Lambda Connector**, you need to have an [AWS Lambda Function](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html), IAM key, and secret pair with permissions for execute function. See the [AWS Lambda developer guide](https://docs.aws.amazon.com/lambda/latest/dg/lambda-permissions.html) to learn more.
```

Comment:
```suggestion
To use the **AWS Lambda Connector**, you need to have an [AWS Lambda Function](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html), IAM key and secret pair with permissions to execute the function. See the [AWS Lambda developer guide](https://docs.aws.amazon.com/lambda/latest/dg/lambda-permissions.html) to learn more.
```

Reply:
@igpetrov @Oleksiivanov, do we need to add this sentence in the Prerequisites?

It is highly recommended not to expose your AWS IAM credentials as plain text but rather use Camunda Secrets. See an [appendix entry](#how-do-i-store-aws-iam-secrets-for-my-lambda-connector) to find out more.

Reply:
@Oleksiivanov this looks good, but it can just be normal text. So remove italics and bold. 
Also last sentence should be "See Manage secrets to find out more."